client/my-sites/email/email-providers-comparison/email-provider-card.jsx

```diff
@@ -42,16 +43,29 @@ function EmailProviderCard( {
 
 	const labelForExpandButton = expandButtonLabel ? expandButtonLabel : buttonLabel;
 
+	const showStar = detailsExpanded && starLabel;
```

Comment:
Nit: I see sometimes we use "should" prefix in this kind of constants. What's the general rule for this?
```suggestion
	const shouldShowStar = detailsExpanded && starLabel;
```

Reply:
I don't think there is a rule for that, at least I'm not aware of any one. I'm not a big fan of this notation for booleans though, as it is longer and doesn't really clarify anything in my opinion. I'll mention that discussion in our Slack channel to see what others think of that.